src/pykeen/experiments/cli.py

```diff
@@ -13,6 +12,9 @@
 
 import click
 
+from pykeen.ablation import prepare_ablation_from_path
```

Comment:
keep big imports inside CLI functions for faster loading times (and use relative imports)

Reply:
Done in a009fde9